communication/src/main/java/org/jboss/da/communcation/pnc/ReadResource.java

```diff
@@ -0,0 +1,25 @@
+package org.jboss.da.communcation.pnc;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+/**
+ * Created by dcheung on 22/06/15.
+ */
+public class ReadResource {
+
+    public static String getResource(String key) {
```

Comment:
IMHO this should be bean and Properties loaded in @PostConstruct method.
I agree with slaskawi that when the properties file doesn't exist, there should be fatal error.